lib/svgo/coa.js

```diff
@@ -47,7 +47,10 @@ export default function makeProgram(program) {
       '-p, --precision <INTEGER>',
       'Set number of digits in the fractional part, overrides plugins params',
     )
-    .option('--config <CONFIG>', 'Custom config file, only .js is supported')
+    .option(
+      '--config <CONFIG>',
+      'Custom config file. Only .js, .mjs, .cjs is supported',
```

Comment:
```suggestion
      'Custom config file, only .js, .mjs, and .cjs is supported',
```

Just to keep things consistent with our other help strings and docs.

Reply:
Ah! Also a fan of the Oxford comma. Not sure how I missed that :D